### ISSUE TITLE
Consistency in Strahlkorper variables documentation

### ISSUE THREAD

**Issue by mar-celine** [sxs-collaboration/spectre#576]
I think we should take some time to carefully go over the definitions we are giving to variables named `n`, `s`, and `S` and make sure they are all consistent with one another. 

e.g. https://sxs-collaboration.github.io/spectre/group__SurfacesGroup.html

**Comment by kidder** [comment]
Use \hat{r} for Rhat, n_i for unnormalized normal one-form, S_i for unit normal one form, r_{surf} for radius of surface as function of \theta,\phi,  x_{surf} for (x,y,z) of surface.  Figure out what n is in the definition of extrinsic curvature, document and check tangents

**Comment by nilsdeppe** [comment]
@markscheel @geoffrey4444 progress update?

**Comment by geoffrey4444** [comment]
I looked over the help for current develop and for the spin magnitude (PR 705). All are internally consistent, and all are consistent with the convention Larry proposed except for spin_function and area_element. Since spin_magnitude in 705 is ok, I’ll do a separate PR to fix.

**Comment by geoffrey4444** [comment]
I also suggest amending the notation so unit normal is s, not S, because S is commonly used for dimensionful spin in the literature, whereas the spin papers I’ve been part of tend to use s^i for unit normal. Please confirm if this is ok, or if you want to rely on help text distinguishing spin S from normal S^i.

**Comment by markscheel** [comment]
I agree with Geoffrey: far as I can tell, the only inconsistency is s^i versus S^i for unit normal. We should choose one. I don't care which one, but we should be consistent.  I slightly prefer s^i because that's what it usually is in papers.

**Comment by kidder** [comment]
I also prefer s^i

**Comment by geoffrey4444** [comment]
OK, if there are no objections, I will make a new PR changing S^i -> s^i and otherwise ensuring everything conforms to Larry's notation in the above comment.

**Comment by nilsdeppe** [comment]
Sounds good to me :)

**Comment by kidder** [comment]
@geoffrey4444 did you do this yet?

**Comment by nilsdeppe** [comment]
@geoffrey4444 is this fixed?
